src/extra/MarkdownEditor/defaultStyle.ts

```diff
@@ -19,7 +19,7 @@ export default css`
 	}
 	.CodeMirror-gutter-filler,
 	.CodeMirror-scrollbar-filler {
-		background-color: transparent;
+		background-color: #fff;
```

Comment:
Can we get some context on why these changes are needed?
Could they be in a separate PR?

Reply:
This is auto generated when running npm run generate-snapshots

Reply:
@JSReds let's then have it as the first commit on its own just to be clear.